src/test/kotlin/org/rust/ide/inspections/lints/RsUnusedImportInspectionTest.kt

```diff
@@ -178,6 +178,21 @@ class RsUnusedImportInspectionTest : RsInspectionsTestBase(RsUnusedImportInspect
         }
     """)
 
+    @MockEdition(CargoWorkspace.Edition.EDITION_2018)
+    @MockAdditionalCfgOptions("intellij_rust")
+    fun `test cfg-disabled import and usage`() = checkByText("""
+        mod foo {
+            pub struct S {}
+        }
+
+        mod bar {
```

Comment:
This test would also succeed if the implementation completely ignored `cfg`, right? Maybe
```rust
#[cfg(not(intellij_rust))]
fn test(_: S) {}
```
should be removed from the test?

Reply:
Indeed, thanks!